Docker_files/ocsci_container/scripts/run-ocs-ci.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+mkdir -p /opt/cluster/logs
+mkdir -p /opt/cluster_path/auth
+cp -R /opt/cluster/* /opt/cluster_path/auth
```

Comment:
Those two lines then you can delete from here.

Reply:
@petr-balogh on line 5, I create a new dir
on line 6, I copy the kubeconfig file to `/opt/cluster_path/auth`